e2e/browser/.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
I think this change in our linting configuration means we don't need this anymore: https://github.com/inrupt/javascript-style-configs/pull/22

Reply:
It looks like the whole e2e folder isn't linted at all. Will fix the config and the associated errors in a separate PR.